staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -307,6 +307,10 @@ func (s *GenericAPIServer) installAPIResources(apiPrefix string, apiGroupInfo *A
 		}
 
 		apiGroupVersion := s.getAPIGroupVersion(apiGroupInfo, groupVersion, apiPrefix)
+		if len(apiGroupVersion.Storage) == 0 {
+			continue
```

Comment:
The 4 lines above do the same, merged in https://github.com/kubernetes/kubernetes/pull/42911/files#diff-8c3b369a56b450db4f2c3715bec58edcR313. So we can drop this PR.